src/pages/events-description/index.js

```diff
@@ -24,25 +25,66 @@ export const EventsPageDescription = () => {
   const [event, setEvent] = useState([])
   const [thumbsSwiper, setThumbsSwiper] = useState(null)
   const [galeria, setGaleria] = useState([])
+  const [activities, setActivities] = useState([])
   const { id } = useParams()
+  const activitiesList = []
   useEffect(() => {
-    cms.get(`api/events/${id}/?populate=parceires&populate=fotos_evento`).then((response) => {
-      const { data } = response.data
-      setEvent(data)
+    cms
+      .get(
+        `api/events/${id}/?populate=parceires
+        &populate=fotos_evento
+        &populate=subeventos
+        &populate=oficinas
+        &populate=oficinas.foto_divulgacao
+        &populate=subeventos.foto_divulgacao`
+      )
+      .then((response) => {
+        const { data } = response.data
+        setEvent(data)
 
-      const fotoDivulgacao = data.attributes.fotos_evento
-      const images = fotoDivulgacao.data.map(
-        (image) => {
+        const fotoDivulgacao = data.attributes.fotos_evento
+        const images = fotoDivulgacao.data.map((image) => {
           return {
             id: image.id,
             name: image.attributes?.name,
             url: process.env.REACT_APP_URL_CMS + image.attributes?.url
           }
-        }
-      )
-      setGaleria(images)
-    })
-  }, [])
+        })
+        setGaleria(images)
+        const oficinas = data.attributes.oficinas.data.map((oficina) => {
+          console.log(oficina)
+          return {
+            id: oficina.id,
+            type: 'workshops',
+            name: oficina.attributes.nome,
+            url:
+            process.env.REACT_APP_URL_CMS +
```

Comment:
Eu fiquei pensando aqui agora: dado que a gente configura o cliente do cms para usar a url do CMS por padrão, eu acho que não precisamos concatenar ela de novo aqui, pois o Axios já vai fazer isso automaticamente. Eu vou testar isso. De repente conseguimos simplificar esse tipo de coisa se isso for verdade.

Reply:
Tiramos a concatenação da linha 61, porém o código quebrou. Aguardamos novas orientações de como prosseguir.

Reply:
Opa! Pode manter a concatenação, eu vou testar isso separadamente com mais calma! :)